docs/blog/newsletter-december-2017.rst

```diff
@@ -0,0 +1,90 @@
+
+Whew! Hopefully that gives you some things to look back on or forward to with excitement!
+
+Please note that the newsletter will be taking January off, to give our editorial crew a little break. We'll be back in the saddle with the February issue. In the meantime, have a happy holiday season and enjoy one last round of 2017 documentation stories, below!
+
+****************************************
+Help your contributors help your project
+****************************************
+
+Contribution guidelines are an important part of any open source project. A discussion about writing good contribution guidelines came up on Slack this month, and yielded some useful pearls of wisdom:
+
+* Contribution guidelines can range form a few pithy sentences to multi-page instructions including a style guide, process instructions, and pull request templates. Small projects will require less detail, but as your project and/or community of contributors grows, it's worth scaling your guidelines accordingly.
+
+* When determining the level of detail to go into, keep in mind how many maintainers you have. Remember that your guidelines will need maintenance just like any other contribution.
+
+* Consider starting with a contribution template when creating your guidelines. Even if you don't have much guidance beyond the template itself, it gives people a place to start. (`Check out this one <https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md>`_ for an example!)
+
+* Think about including code of conduct as part of your contribution guidelines. It's helpful to set expectations about conduct as early as possible, and a clear and visible code of conduct can help your community grow the way you want it to.
+
+If you're looking for inspiration, this `explanation of Node.js's contribution policy (2016) <https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951>`_ is a great place to start.
+
+****************************
+It's just documentation, man
+****************************
+
+An interesting discussion cropped up on `Hacker News <https://news.ycombinator.com/item?id=15779382>`_ this month, about "man pages" (short for command line user **man**uals). The story made the rounds in a couple of Write the Docs spaces and sparked some lively conversations.
+
+In the original HN conversation, diverse opinions were expressed on a range of subjects. For example, should man pages include tutorial type information? Is the traditional structure of Synopsis, Description, Options, and Examples at all useful? Should command-line parameters should be sorted or grouped? Do you need to be able to jump to a particular command-line option without resorting to wildcard search?
+
```

Comment:
Suggest "tutorial-type" with hyphen

Reply:
Also, "Should command-line parameters be sorted or grouped?" (remove second "should")

Reply:
👍